### DISCUSSION TITLE
meta.code vs meta.constant

### DISCUSSION THREAD

**Discussion by nschloe** [keygen-sh/community#66] [Q&A] [answered]
Sometimes, error codes are given in `meta.code`, other times in `meta.constant`. What's that about?

**Comment by ezekg** [comment, marked as answer]
I take it you’re referring to validation codes, not error codes. You’re likely using multiple accounts that have different [pinned API versions](https://keygen.sh/docs/api/versioning/#pinning). In API v1.2, we renamed `meta.constant` to `meta.code` (see [changelog](https://keygen.sh/changelog/) entry for July 2022). When we make a breaking API change, we do it in a “backwards compatible” way via versioning (further detailed in our API versioning docs).

I’d recommend pinning your integration to a specific API version via the `Keygen-Version` header. For example:

```
Keygen-Version: 1.3
```

This will force your requests to use a specific version, regardless of what the account’s pinned version is.
